### PR TITLE
changed logger debug mode to log level

### PR DESCRIPTION
**Pull request by RoyUP9**

### CODE REVIEW

File: shared/consts.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ const (
 	ConfigFileName                   = "mizu-config.json"
 	GoGCEnvVar                       = "GOGC"
 	DefaultApiServerPort             = 8899
-	DebugModeEnvVar                  = "MIZU_DEBUG"
+	LogLevelEnvVar                   = "MIZU_LOG_LEVEL"
```
Comment (IgorGov) [inline shared/consts.go:16]: LOG_LEVEL

File: cli/config/configStruct.go [hunk 2]
```diff
@@ -32,6 +33,15 @@ type ConfigStruct struct {
 	KubeConfigPathStr      string                      `yaml:"kube-config-path"`
 	ConfigFilePath         string                      `yaml:"config-path,omitempty" readonly:""`
 	HeadlessMode           bool                        `yaml:"headless" default:"false"`
+	LogLevelStr            string                      `yaml:"server-log-level,omitempty" default:"INFO" readonly:""`
```
Comment (IgorGov) [inline cli/config/configStruct.go:36]: log-level

File: cli/cmd/root.go [hunk 3]
```diff
@@ -23,6 +23,11 @@ Further info is available at https://github.com/up9inc/mizu`,
 		if err := config.InitConfig(cmd); err != nil {
 			logger.Log.Fatal(err)
 		}
+
+		if err := config.Config.Validate(); err != nil {
```
Comment (gadotroee) [inline cli/cmd/root.go:27]: why not validate when Init?

Comment (RoyUP9) [reply cli/cmd/root.go:27]: because you have to do it after the config is initiated

**Review by IgorGov — APPROVED** [review]

**Pull request merged**